packages/archetype/archetype.0.1.1/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+maintainer: "The Archetype development team <archetype-dev@edukera.com>"
+authors: ["Benoit Rognier <benoit.rognier@edukera.com>"
+          "Guillaume Duhamel <guillaume.duhamel@edukera.com>"
+          "Pierre-Yves Strub <pierre-yves.strub@polytechnique.edu>"]
+homepage: "https://github.com/edukera/archetype-lang"
+bug-reports: "https://github.com/edukera/archetype-lang/issues"
+dev-repo: "git+https://github.com/edukera/archetype-lang.git"
+license: "MIT"
+
+build: [
+  ["dune" "build"]
+]
+
+depends: [
```

Comment:
Why are all your dependencies that strict? Could you make them less strict please (using `>=` instead of `=`). Otherwise very few people will actually be able to use your package.

Reply:
yes, you're right, there is no reason to be so strict, I have committed the fix